pycbc/detector.py

```diff
@@ -445,3 +445,67 @@ def time_delay_from_earth_center(self, right_ascension, declination, t_gps):
             np.array([np.float32(earth.x), np.float32(earth.y),
                       np.float32(earth.z)]), right_ascension,
             declination, t_gps)
+
+    def __h_ij__(self, X):
+        # pos return hij
+        return np.random.random(3)
```

Comment:
this will return a set of 3 values: having 2 values, +,x polarization and h (sky location)